go.mod

```
@@ -0,0 +1,10 @@
+module github.com/ratify-project/ratify-cli/v2
+
+go 1.23.0
```

go 1.23.1 has some security issues. Let's make it go 1.23.1.
```suggestion
go 1.23.1
```

---

Updated